Java/InsertionSort.java

```diff
@@ -0,0 +1,41 @@
+/* Insertion Sort using Iteration */
+
+public class InsertionSort{
+ 
+     public static void insertionSort(int[] ar){
+       
+    //Starting index from index 1 to create sorted array at left side of the array
+
+             for(int i = 1; i < ar.length; i++){
+
+                 int j = i-1,k = i;                // Assigning j = i-1 and iterating to 0 in sorted part of the array
+                 while(j >= 0 && ar[k] < ar[j]){   // checking if element at index i is smaller than elements in the sorted array and then swap them
+                     swap(ar,k,j);                 // swapping elements in the same array by passing ar and indexes i and j
+                     j--;
+                     k--;
+                 }
+
+             }
+
```

Comment:
Please remove unnecessary spaces and indentations. Please use a Code Formatter to format your code.